ray/README.md

```diff
@@ -1,68 +1,83 @@
 
-## Components  of the Ray deployment
 
-1. [Ray operator](./operator/base/ray-operator-deployment.yaml): The operator will process RayCluster resources and schedule ray head and worker pods based on requirements. Image Containerfile can be found [here](https://github.com/thoth-station/ray-operator/blob/master/Containerfile). 
-2. [Ray CR](./operator/base/ray-custom-resources.yaml):  RayCluster Custom Resource (CR) describes the desired state of ray cluster.
-3. [Ray Cluster](./cluster/base/ray-cluster.yaml): Defines an instance of an example Ray Cluster. Image Containerfile can be found [here](https://github.com/thoth-station/ray-ml-worker/blob/master/Containerfile) 
- 
+## Components of the KubeRay deployment
 
-## Deploy the RayCluster Components:
+1. Namespace 'ray-system'
+2. Custom Resource Definitions
+3. Operator
+4. Roles and Service Accounts
 
-Prerequisite to install RayCluster with ODH:
+## Installation
 
-* Cluster admin access
-* An ODH deployment
-* [Kustomize](https://kustomize.io/) 
+There are two ways to install KubeRay. You can either **install it directly into your cluster alongside an existing Open Data Hub deployment using Kustomize** OR you can **include it as an additional component in your Open Data Hub KfDef**. Below we will walk through both approaches. 
 
-### Install Ray
 
-We will use [Kustomize](https://kustomize.io/) to deploy everything we need to use Ray with Open Data Hub. 
+## Install via Kustomize:
 
-#### Install the operator and custom resource 
-
-First use the `oc kustomize` command to generate a yaml containing all the requirements for the operator and the "raycluster" custom resource, then `oc apply` that yaml to deploy the operator to your cluster. 
+This method requires that you have [kustomize](https://kustomize.io/) and the [oc client](https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html) installed in your environment. Then, from within the `ray/` directory of this repo you can run the following:
 
 ```bash
-$ oc kustomize ray/operator/base > operator_deployment.yaml
+cd operator/base
+oc kustomize > deploy_kuberay.yaml
+oc create -f deploy_kuberay.yaml
 ```
-```bash
-$ oc apply -f operator_deployment.yaml -n <your-namespace>
+
+## Install via KfDef:
+
+Alternatively, you can include the KubeRay components in your ODH KfDef. This will ensure that all of the components above get installed as part of your ODH deployment.  
+
+```yaml
+ # KubeRay
+  - kustomizeConfig:
+      repoRef:
+        name: manifests-contrib
+        path: ray/operator
+    name: ray-operator
 ```
+You can find a minimal [KfDef here](ray-minimal-kfdef.yaml), that can be used to just install the core ODH components, along with everything you'll need to use KubeRay.  
+
+
+
+
+## Confirm the operator is running:
+
+Once installed, you can confirm the KubeRay operator has been deployed correctly with `oc get pods -n ray-system`.
```

Comment:
These instructions don't work when using the minimal kfdef. There is no ray-system namespace created. If this is supposed to be the opendatahub namespace, there is a pod called `ray-operator-...`, but not kuberay.